cli/src/install.sh

```diff
@@ -23,6 +23,11 @@ banner() {
     printf "\n    %bphylum-cli%b installer\n\n" "${GREEN}" "${NC}"
 }
 
+cross_sed() {
+    sed -i.bak "$1" "$2"
+    rm "${2}.bak"
```

Comment:
```suggestion
    sed -i'.bak' "$1" "$2"
    rm -f "${2}.bak"
```
The single quotes appear to be necessary...at least from the [blog post](https://singhkays.com/blog/sed-error-i-expects-followed-by-text/) used to confirm this behavior as a fix.

The `rm -f` will ensure no prompts for confirmation.

Reply:
Huh, after looking at that blog post I'm going to just try `sed -i'' -e` before merging this. Good find, I had searched a bit for a simple incantation like that but came up dry.

Reply:
FWIW...I am on a M1 Mac and tried the `sed -i''` method and still had issues...but it is definitely possible I missed something. Trying whatever method is chosen on a non-macOS system is the other piece of this puzzle...don't want to break it for those users instead.

Reply:
Ah, I thought this worked at first, but it ends up leaving a `~/.zshrc-e` file.

Reply:
To test this I've been alternating between having `gsed` preferred (`PATH="/opt/homebrew/opt/gnu-sed/libexec/gnubin:$PATH"`) vs the mac default BSD preferred (`PATH="/usr/bin:$PATH"`) but yea a quick sanity check on a linux box would be good.